### PR TITLE
Yet Another Admin Tweaks PR

### PR DESCRIPTION
**Pull request by DominikPanic**

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -287,14 +287,14 @@
 					to_chat(usr, "<span class='warning'>Arrays recycling.  Please stand by.</span>")
 					return FALSE
 
-				var/input = stripped_input(usr, "Please choose a message to transmit to the TGMC High Command.  Please be aware that this process is very expensive, and abuse will lead to termination.  Transmission does not guarantee a response. There is a small delay before you may send another message. Be clear and concise.", "To abort, send an empty message.", "")
-				if(!input || !(usr in view(1,src)) || authenticated != 2 || world.time < cooldown_central + COOLDOWN_COMM_CENTRAL)
+				var/msg = input(usr, "Please choose a message to transmit to the TGMC High Command.  Please be aware that this process is very expensive, and abuse will lead to termination.  Transmission does not guarantee a response. There is a small delay before you may send another message. Be clear and concise.", "To abort, send an empty message.", "")
+				if(!msg || !(usr in view(1, src)) || authenticated != 2 || world.time < cooldown_central + COOLDOWN_COMM_CENTRAL)
```
Comment (Ghommie) [inline code/game/machinery/computer/communications.dm]:
```suggestion
				if(!msg || !usr.Adjacent(src) || authenticated != 2 || world.time < cooldown_central + COOLDOWN_COMM_CENTRAL)
```

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 2]
```diff
@@ -212,6 +212,8 @@
 	victim.death() // Certain species were still surviving bursting (predators), DEFINITELY kill them this time.
 	victim.chestburst = 2
 	victim.update_burst()
+	log_combat(src, src, "bursted as a [src].")
+	log_game("[key_name(src)] bursted as a [src] at [AREACOORD(src)].")
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm]: "chestbursted" would be a little more descriptive.

File: code/_onclick/hud/screen_objects.dm [hunk 3]
```diff
@@ -124,6 +124,8 @@
 	var/selecting = "chest"
 
 /obj/screen/zone_sel/update_icon(mob/living/user)
+	if(!user)
```
Comment (spookydonut) [inline code/_onclick/hud/screen_objects.dm]: How is this admin related? Also if this is called without a valid user something is seriously wrong

File: code/modules/admin/verbs/player_notes.dm [hunk 4]
```diff
@@ -273,7 +301,7 @@
 	var/savefile/info = new("data/player_saves/[copytext(key, 1, 2)]/[key]/info.sav")
 	var/list/infos
 	info >> infos
-	if(!infos || !infos.len)
+	if(!infos || !length(infos))
```
Comment (spookydonut) [inline code/modules/admin/verbs/player_notes.dm]:
```suggestion
	if(!length(infos))
```
length(null) returns 0

File: code/modules/admin/verbs/pray.dm [hunk 5]
```diff
@@ -36,7 +36,7 @@
 		to_chat(usr, "Your prayers have been received by the gods.")
 
 
-/proc/Centcomm_announce(var/text , var/mob/sender , var/iamessage)
+/proc/tgmc_message(text, mob/sender)
 	text = copytext(sanitize(text), 1, MAX_MESSAGE_LEN)
 	for(var/client/C in GLOB.admins)
 		if(check_other_rights(C, R_ADMIN, FALSE))
```
Comment (spookydonut) [inline code/modules/admin/verbs/pray.dm]: SEND_SOUND(C, 'sound/effects/sos-morse-code.ogg')

File: code/modules/mob/living/living_verbs.dm [hunk 6]
```diff
@@ -298,14 +298,16 @@
 		resting = TRUE
 		to_chat(src, "<span class='notice'>You are now resting.</span>")
 		update_canmove()
-	else if(action_busy) // do_after is unoptimal
+	else if(action_busy)
 		return
```
Comment (spookydonut) [inline code/modules/mob/living/living_verbs.dm]: to_chat(src, "you are still in the process of standing up")

File: code/modules/mob/login.dm [hunk 7]
```diff
@@ -33,6 +33,8 @@
 	if(!hud_used) create_hud()
 	if(hud_used) hud_used.show_hud(hud_used.hud_version)
 
+	log_message("has logged in.", LOG_OOC, "blue")
```
Comment (spookydonut) [inline code/modules/mob/login.dm]: this will trigger everytime someone ghosts.

File: code/game/atoms.dm [hunk 8]
```diff
@@ -633,5 +633,8 @@ Proc for attack log creation, because really why not
 /atom/vv_get_dropdown()
 	. = ..()
 	. += "---"
+	var/turf/curturf = get_turf(src)
+	if(curturf)
+		.["Jump to"] = "?_src_=holder;[HrefToken()];observecoordjump=1;X=[curturf.x];Y=[curturf.y];Z=[curturf.z]"
```
[the remark this answers is not in the document]
Comment (DominikPanic) [reply code/game/atoms.dm]: This is for any atom that you can get through with VV, it's an additional way for admins to move effectively along their other commands.

[the remark this answers is not in the document]
Comment (spookydonut) [reply code/game/atoms.dm]: i mean a code macro for the x/y/z stuff
